base/compiler/ssair/ir.jl

```diff
@@ -351,54 +351,52 @@ getindex(it::UseRefIterator) = it.use[1].stmt
 #    use::Int
 #end
 
-struct OOBToken
-end
-
-struct UndefToken
-end
-const undef_token = UndefToken()
+struct OOBToken end
+const OOB_TOKEN = OOBToken.instance
```

Comment:
Just for my own edification: What's the benefit of using `OOBToken.instance` versus `OOBToken()`? They should be equivalent, right?

Reply:
Yeah, they should be really equivalent both in terms of behavior and performance.

Reply:
Okay, thanks. Out of curiosity then why opt for `.instance` in this case?

Reply:
Yeah, I think just using `OOBToken()` is canonical. I don't necessarily see why we need a constant either,

Reply:
Okay, let me remove those constants.